### PR TITLE
[RW-7066][risk=medium] Launch enforce RAS linking 

### PR DESCRIPTION
**Pull request by yonghaoy**
Description:

- Launch RAS linking in all env. 
- Disable sync access cron job in prod
- Move user-access to a dedicate page. 

Some analysis:

- test: all users were bypassed
- perf: all users were bypassed
- staging: all users were bypassed
- stable: all users were bypassed
- preprod: I will not turn it off. 

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java [hunk 1]
```diff
@@ -511,6 +511,7 @@ public void test_DuccBecomesOutdated() {
     // bypass the other access requirements
     final DbUser dbUser = userDao.findUserByUserId(userId);
     accessModuleService.updateCompletionTime(dbUser, AccessModuleName.ERA_COMMONS, TIMESTAMP);
+    accessModuleService.updateCompletionTime(dbUser, AccessModuleName.RAS_LOGIN_GOV, TIMESTAMP);
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:514]: why is this the only test which needs to be changed?  I would expect many tests to fail

Comment (yonghaoy) [reply api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:514]: seems that is the only test that checks user tier.

File: ui/src/app/components/side-nav.tsx [hunk 2]
```diff
@@ -336,6 +340,15 @@ export const SideNav = (props: SideNavProps) => {
             active={userAdminActive()}
           />
         )}
+      {hasAuthorityForAction(profile, AuthorityGuardedAction.USER_ADMIN) &&
+        showAdminOptions && (
+          <SideNavItem
+            content={'User Access'}
```
[the remark this answers is not in the document]
Comment (yonghaoy) [reply ui/src/app/components/side-nav.tsx:346]: Screen Shot 2022-01-28 at 11 39 17 AM
Change to bulk sync user access? Other wise, it's bit too long for a side bar menu

File: ui/src/app/components/side-nav.tsx [hunk 3]
```diff
@@ -115,6 +115,10 @@ const userAdminActive = () => {
   return window.location.pathname.startsWith('/admin/user');
 };
 
+const userAccessActive = () => {
+  return window.location.pathname.startsWith('/admin/user-access');
```
Comment (jmthibault79) [inline ui/src/app/components/side-nav.tsx:119]: bulk-sync-user-access

File: ui/src/app/components/side-nav.tsx [hunk 4]
```diff
@@ -336,6 +340,15 @@ export const SideNav = (props: SideNavProps) => {
             active={userAdminActive()}
           />
         )}
+      {hasAuthorityForAction(profile, AuthorityGuardedAction.USER_ADMIN) &&
+        showAdminOptions && (
+          <SideNavItem
+            content={'Bulk Sync of User Access'}
```
Comment (jmthibault79) [inline ui/src/app/components/side-nav.tsx:346]: New name LGTM!

File: api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java (hunk 1 above)
Comment (jmthibault79) [reply api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java:514]: I was thinking of UserServiceAccessTest - but I see now that this flag is already true there.  👍

**Review by jmthibault79 — APPROVED** [review]

**Review by calbach — APPROVED** [review]

**Pull request merged**